### ISSUE TITLE
should i move the the rtw89 repo?

### ISSUE THREAD

**Issue by saikatm** [lwfinger/rtw8852be#34]
as the new commit asks to use the rtw89 repo..
so I should we move to the new repo? and why this repo will be redundant ?

**Comment by lwfinger** [comment]
Yes, you should. This repo contains a driver that no one is supporting. The one in rtw89 matches what will be in the 6.2 kernel, and is the one supported by Realtek in the kernel, and me for users of older kernels.

You can continue with this code indefinitely, but do not expect me to pay any attention to any issues after Dec. 1.

**Comment by saikatm** [comment]
I tried the rtw89 repo but it did now work.
this repo with latest commit did work. thanks alot.
